static/third/marked/lib/marked.js

```diff
@@ -863,6 +870,20 @@ InlineLexer.prototype.usermention = function (username, orig) {
   return orig;
 };
 
+InlineLexer.prototype.streamlink = function(stream_name, orig) {
+  if (typeof this.options.streamLinkHanlder !== 'function')
+  {
+    return orig;
+  }
+
+  var handled = this.options.streamLinkHanlder(stream_name);
+  if (handled !== undefined) {
+    return handled;
+  }
+
+  return orig;
+}
```

Comment:
Somewhere this integration use `stream_data.get(stream_name)`  to check that the stream actually exists.